src/Extensions/Hosting.Services.Remoting/Remoting.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.ServiceFabric.Services.Remoting.FabricTransport.Runtime;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Remoting
+{
+	/// <summary>
+	/// Static methods to simplify creation of <see cref="FabricTransportRemotingListenerSettings" />
+	/// </summary>
+	public static class Remoting
+	{
+		/// <summary>
+		/// Creates <see cref="FabricTransportRemotingListenerSettings" /> with provided EndpointResourceName
+		/// </summary>
+		public static FabricTransportRemotingListenerSettings EndpointResourceName(string value)
```

Comment:
>EndpointResourceName [](start = 56, length = 20)

I'd suggest calling this `ToEndpointResourceName` for alignment with similar methods

Reply:
I've changed name to 'RemotingSettings.WithEndpoint()'